thousandeyes/data_source_thousandeyes_agent.go

```diff
@@ -14,20 +14,15 @@ func dataSourceThousandeyesAgent() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 			"agent_name": {
-				Type:     schema.TypeString,
-				Required: true,
+				Type:        schema.TypeString,
+				Required:    true,
 				Description: "The name of the agent.",
 			},
 			"agent_id": {
-				Type:     schema.TypeInt,
-				Computed: true,
+				Type:        schema.TypeInt,
+				Computed:    true,
 				Description: "The unique ID of the agent.",
 			},
-			"agent_type": {
-				Type:     schema.TypeString,
-				Computed: true,
-				Description: "The type of agent.",
-			},
```

Comment:
Not needed, this gets cleared here: https://github.com/thousandeyes/terraform-provider-thousandeyes/blob/main/thousandeyes/util.go#L164